awswrangler/catalog/_create.py

```diff
@@ -501,16 +505,19 @@ def create_database(
     ...     name='awswrangler_test'
     ... )
     """
-    args: Dict[str, str] = {}
     client_glue: boto3.client = _utils.client(service_name="glue", session=boto3_session)
-    args["Name"] = name
+    args: Dict[str, str] = {"Name": name}
     if description is not None:
         args["Description"] = description
 
-    if catalog_id is not None:
```

Comment:
Why is this branch removed?

Reply:
By using https://github.com/awslabs/aws-data-wrangler/blob/main/awswrangler/catalog/_utils.py#L17 in the `create_database` call, the catalog id is used for the creation if it is given but not used otherwise. Thereby the check is not required but included directly in the create call.

Reply:
Hmm I figured so but wanted to double check with you. Thanks @maxispeicher :)